src/components/Grid/GridLegend.js

```diff
@@ -3,7 +3,7 @@ import React from "react";
 
 import styled from "styled-components";
 
-function GridLegend() {
+function GridLegend({ bonusLabel = "Bonus" }) {
```

Comment:
`bonusLabel` is not used anywhere. Can we remove it?

Reply:
Hmm, good catch!

Reply:
Thanks again!

Reply:
Wow!